### PR TITLE
Tattoo LED

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary

```SUMMARY: Content "Adds new CBM : LED Tattoo"```
#### Purpose of change

Tired of being a dull looking survivor ? Looking to become part of the cool kids ?
Well, look no more ! With this brand new flashy (glowy*) CBM you'll be able to show them how unique you are !
#### Describe the solution

Adds LED Tattoo : a easy to install CBM that glows when activated, can serve as a night light.
Adds accompanying faulty CBM : Glowy Thing a faulty CBM that randomly makes you glow.
#### Describe alternatives you've considered

None, useless CBMs are obviously a must have.
#### Additional context

The goal here is to add a CBM that is more useful for RP than anything else. 
Its glowy function could be useful early game, so maybe it could be serve for a new profession ; `Bionic_Raver` (??)
It's easy to install but should not break any game balance.

### CODE REVIEW

File: src/player.cpp [hunk 1]
```diff
@@ -6042,6 +6044,10 @@ void player::suffer()
         body_part bp = random_body_part(true);
         add_effect( effect_formication, 10_minutes, bp );
     }
+    if (has_bionic ( bio_glowy ) && one_in(500) !has_effect( effect_glowing )) {
+        add_msg_if_player(m_bad, _("Your malfunctioning bionic starts to glow!"));
```
Comment (mlangsdorf) [inline src/player.cpp]:
```suggestion
    if( has_bionic( bio_glowy ) && !has_effect( effect_glowing ) && one_in( 500 ) ) {
        add_msg_if_player( m_bad, _( "Your malfunctioning bionic starts to glow!" ) );
```

Did your original version even compile?  Please compile and test your changes before pushing them.

**Comment by Fris0uman** [comment]
Sorry about that, it should be fixed now. I managed to compile it and test the CBM and it seems to work.
Don't know how to test the faulty one though

**Comment by DracoGriffin** [comment]
> 
> 
> Sorry about that, it should be fixed now. I managed to compile it and test the CBM and it seems to work.
> Don't know how to test the faulty one though

Make a temporary profession that starts with it, similar to Broken Cyborg.

**Comment by Fris0uman** [comment]
So turns out `effect_glowing` is the effect induced by glowing goo so that's not what we want for `Glowy Thing` as it can be wahsed by water and displays a message about being covered in thick goo.

**Comment by Fris0uman** [comment]
Now everything seems to work.
Should I remove `name` and `description` from glowy_led effect so that it doesn't clutter @ menu (its redundant with `Lit-up`) ?

File: data/json/bionics.json [hunk 2]
```diff
@@ -412,6 +412,23 @@
     "time": 25,
     "flags": [ "BIONIC_TOGGLED" ]
   },
+  {
+    "id": "bio_tattoo_led",
+    "type": "bionic",
+    "name": "LED Tattoo",
+    "description": "A simple LED display is implanted under the skin, bright enough to be seen through the dermal layer when active, vanishing from sight when inactive. You might not be able to show off in clubs with it anymore but its faint glow could provide some confort in the dark.",
```
Comment (matskuman5) [inline data/json/bionics.json]:
```suggestion
    "description": "A simple LED display is implanted under the skin, bright enough to be seen through the dermal layer when active, vanishing from sight when inactive. You might not be able to show off in clubs with it anymore but its faint glow could provide some comfort in the dark.",
```

**Comment by Fris0uman** [comment]
Lit-up has "max_duration": 1, does that mean that `add_effect` can only  apply it for only 1 turn ?

**Comment by Fris0uman** [comment]
I have no idea why it fails. It seems that travis ran out of time ...

**Comment by Fris0uman** [comment]
So how do I resolve the conflict here ? I'm not even sure what's conflicting as it seem to higlight every line ^^"

Also this has been sitting here for about 2 weeks, is it juste because of it's very low priority or is this PR tacitly rejected ?

**Comment by Night-Pryanik** [comment]
> So how do I resolve the conflict here ?

Press this button:

default

>Also this has been sitting here for about 2 weeks, is it juste because of it's very low priority or is this PR tacitly rejected ?

Queue.

**Comment by Fris0uman** [comment]
Got it 👍

**Comment by Marrim** [comment]
Having this active should add morale if you have the Stylish trait.

**Comment by Fris0uman** [comment]
Yes that seems to make sens, but bionics effects are mostly harcoded so it's not going to be very easy to add.
I will take a look at it anyway, but implementing `fancy` flag for cbm might be too big for this PR.

**Pull request merged**